src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1257,12 +1257,18 @@ Please specify one of them using the `--source` option to proceed.</value>
     <value>This package's version number cannot be determined. To upgrade it anyway, add the argument --include-unknown to your previous command.</value>
     <comment>{Locked="--include-unknown"}</comment>
   </data>
-  <data name="UpgradeUnknownCount" xml:space="preserve">
-    <value>packages have version numbers that cannot be determined. Use "--include-unknown" to see all results.</value>
+  <data name="UpgradeUnknownVersionCount" xml:space="preserve">
+    <value>package(s) have version numbers that cannot be determined. Use "--include-unknown" to see all results.</value>
     <comment>{Locked="--include-unknown"} This string is preceded by a (integer) number of packages that do not have notated versions. </comment>
   </data>
-  <data name="UpgradeUnknownCountSingle" xml:space="preserve">
-    <value>package has a version number that cannot be determined. Use "--include-unknown" to see all results.</value>
-    <comment>{Locked="--include-unknown"} This string is preceded by a (integer) number of packages that do not have notated versions.</comment>
+  <data name="IncludeExplicitArgumentDescription" xml:space="preserve">
+    <value>Upgrade packages that are pinned by default</value>
+  </data>
+  <data name="UpgradeRequireExplicitCount" xml:space="preserve">
+    <value>package(s) is pinned by default and needs to be explicitly upgraded. Use "--include-explicit" to see all results.</value>

```

Comment:
Maybe we shouldn't call out `by default` and just treat the manifest property and pinning as the same feature (to the user).  Internally we can still track the difference so that we can allow unpinning something that is marked as explicit, but externally we just treat it no differently than if the user had requested pinning.